cmake/sysbuild/suit.cmake

```diff
@@ -0,0 +1,347 @@
+#
```

Comment:
Some of the functions from this file are needed by the sysctrl and secdom build systems - which do not use sysbuild and are not aware of the `sysbuild_get` function. Although the two changes I've suggested hep, it does not seem to be clean to include the whole `suit.cmake` file from the sysctrl and secdom build systems.

Maybe we should create a separate file with such SUIT-utility functions, so that this file can be included by the build-systems not using sysbuild. Functions which could be extracted to this file are:
`suit_copy_artifact_to_output_directory`
`suit_render_template`
`suit_create_envelope`

Reply:
Good idea, I'll look into that.

Reply:
Added on the separate [commit](https://github.com/nrfconnect/sdk-nrf/pull/14689/commits/e14c2cb76f3ec2844e619502091dc9154cc7d4c0), please check if this fit sysctr/secdom needs.